src/Orchard.Specs/Blogs.feature

```diff
@@ -269,12 +281,12 @@ Scenario: I can create browse blog posts on several pages
     Then I should see "<h1[^>]*>.*?My Blog.*?</h1>"
         And I should see "<h1[^>]*>.*?My Post 12.*?</h1>"
         And I should see "<h1[^>]*>.*?My Post 11.*?</h1>"
-        And I should not see "<h1[^>]*>.*?My Post 10.*?</h1>"
+        And I should not see "My Post 2"
```

Comment:
This was a false positive, because "I should not see" uses a string contains check (vs. "I should see" is a regex match) and the value checked was also incorrect (since "My Post 10" should appear on the first page).